Wallet fee options

tolgahan-arikan opened the pull request.

SamueleA reviewed (commented):
This is looking great!

![image](https://github.com/user-attachments/assets/1be70dac-9191-48e4-9170-4646c8b52170)

Just two nitpicks:

1. The "Refresh Balance" button should be removed IMO

We should take inspiration from https://sequence.app and streamline this. I think it's perfectly fine to just fetch the fee options and balances upon initially entering the view. If we want them to update in real time in case the user sends currency to his wallet we could have the data being fetched by a react-query hook with a 15-30 second staleTime for the cache.

2. I'm noticing that the back button at the top is disabled in favor of the cancel button. I think we can go all the way and completely hide that back button since it serves no purpose.

corbanbrook reviewed (changes requested):
- Add TokenImage for Fee options
- Just use a borderWidth="thick" and borderColor="active" (i think) to specify selected option rather than gradient purple

tolgahan-arikan commented:
@SamueleA I removed the refresh button + added react-query to fetch balances every 10 seconds, I wanted to keep the button opacity like this instead of removing, otherwise UI looks a bit _jumpy_ :D

@corbanbrook I added the image + border thingy

latest

https://github.com/user-attachments/assets/4cd27253-91d1-489e-923e-6630771909fc

also updated https://kit-qa.pages.dev/
